### DISCUSSION TITLE
Support Water Meter DIEHL HRL-C-G3 ?

### DISCUSSION THREAD

**Discussion by auriou** [wmbusmeters/wmbusmeters#722] [Q&A]
Hello 

My water meter is equipped with this transmitter : DIEHL HRL-C-G3

It is not in the list, is it not compatible with the software.

The link to the transmetter : https://www.diehl.com/metering/nl/producten-en-diensten/systeem-software/systeem-softwarecomponenten/hrl-c-g3-at/576545/

Thx.

**Comment by weetmuts** [comment]
You need to collect a decrypted telegram. Or provide an encrypted telegram and key.

Run for example: wmbusmeters --logtelegrams auto:t1 
to collect encrypted telegrams.

If you have the key do: wmbusmeters  --logtelegrams auto:t1 MyMeter iperl YourMeterId YourMeterKey
The driver iperl does not matter it should decrypt the telegrams fine anyway. 

Open an issue and provide the telegrams there. :-)

**Comment by weetmuts** [comment]
This one seems to be lora? Or is the device configured to transmit wmbus on 868 MHz?

**Reply by auriou** [reply]
I don't know at all, on the site documentation they talk about Lora.  this type of meter seems to be new and common, for new home construction.

The HRLc G3 radio module integrates LoRaWAN on 868 MHz
